src/graphql/helpers.ts

```diff
@@ -100,8 +100,23 @@ export function buildWhereQuery(fields, alias, where) {
 
     const fieldIn = where[`${field}_in`] || [];
     if (fieldIn.length > 0) {
-      query += `AND ${alias}.${field} IN (?) `;
-      params.push(fieldIn);
+      // Handle choice_in json keys search
+      if (alias === 'v' && field === 'choice') {
+        params.push(fieldIn);
```

Comment:
If someone requests with `choice: '1'` it works for single choice voting only but if `choice_in: ['1']` it works for all voting types. 

Both should be similar, they are just different inputs, outputs should be same

Reply:
Otherwise looks good 👍

Reply:
choice is exact match where query, you mean when user search with choice: '1' we convert it to choice_in: ['1']?
Maybe we can remove this exact search query due to complex search of json choices?

Reply:
I am more confused now 😆 

`choice_in` should be the same as `choice`, choice_in just that we pass more inputs

1. If we need an exact match, then `choice_in` should also do an exact match too, but these won't be useful for UI right? works only for single choice

2. Maybe we can have a new option `choice_matches` like before but which can search the JSON for both exact match and JSON choice match and leave `choice` and `choice_in` for an exact match

Reply:
So like it was from the start with choice_in_json_keys? 😃

Reply:
hehe yea but `choice_in_json_keys` was just for choice inside JSON keys, we should also check for an exact match, the name also confuses 🙈  maybe `choice_match` or something like that should be good i think

Reply:
Maybe name it ```choice_contains_path``` after JSON_CONTAINS_PATH ?

Reply:
that name mean it won't check for an exact match 🙈

Reply:
Now we have:
choice - search for exact match (does not work for json choice)
choice_in - search for keys in json OR match in array
How do want to change that?
choice - search for exact match (does not work for json choice)
choice_in - search for match in array (does not work for json choice)
choice_matches - search for keys in json OR match in array